### ISSUE TITLE
Dynamically load snippets. Load only one.

### ISSUE THREAD

**Issue by st0le** [kichik/whypy3#2]
Currently, it embeds all the snippets on the main page. The size of the page will increase and with it the load time when there are more contributions.

**Comment by kichik** [comment]
Agreed. For now it's small enough and GitHub Pages supports gzip. But once it gets bigger, we should probably switch to one or maybe a batch of 10 at a time.
